tapiriik/local_settings.py.example

```diff
@@ -7,6 +7,9 @@ DROPBOX_FULL_APP_SECRET = "####"
 DROPBOX_APP_KEY = "####"
 DROPBOX_APP_SECRET = "####"
 
+GOOGLEDRIVE_CLIENT_ID = "####"
```

Comment:
Good question.  You can add multiple google APIs to the same project and make it a single confirmation from the user - so I guess it depends whether you want to tie the two of them together or not.  If you make them separate projects and allow people to hook up just one they'll have different client ID/client secret pairs.  The other option would just be to have one "Google" sync option and then make the fit/drive question user-configurable I guess.